src/pages/why.tsx

```diff
@@ -0,0 +1,412 @@
+                                    before paying anything at all.
+                                </p>
+
+                                <h2 id="posthog-just-does-the-right-thing">PostHog just does the right thing</h2>
+                                <p>
+                                    We polled our customers - one of the top four reasons they recommend PostHog is
+                                    "developer brand". To us that means we do the right thing for our customers:
+                                </p>
+                                <ul>
+                                    <li>
+                                        <Link to="/blog/analytics-pricing">We roll out pricing decreases</Link> to our
+                                        entire existing customer base
+                                    </li>
+                                    <li>
+                                        We have{' '}
+                                        <Link href="/handbook/growth/sales/refunds#unexpected-stardom">
+                                            side hustle insurance
+                                        </Link>{' '}
+                                        - we refund if your thing goes viral by accident on HackerNews
+                                    </li>
+                                    <li>
+                                        We've got{' '}
+                                        <Link href="https://github.com/posthog/posthog" external>
+                                            a huge open source project
+                                        </Link>{' '}
+                                        (&gt;20k stars), and beyond this you can inspect all our code in our repo so you
+                                        can see exactly what we're doing with your data
+                                    </li>
+                                    <li>
+                                        <Link href="/sales">We don't do outbound sales</Link> - we'd rather spend our
+                                        time on more products
+                                    </li>
+                                    <li>
+                                        You can <Link href="/pricing">see all our pricing publicly</Link> - other people
+                                        don’t hide it because "iT’s a bEtTeR vAlue!"
+                                    </li>
+                                    <li>
+                                        We don’t sell products at a loss - we run the company default alive. This means
+                                        we’ve never had a layoff, and we don’t take huge risks that means we could
+                                        disappear overnight.
+                                    </li>
+                                    <li>We don’t screw you on contract terms like auto-renewal.</li>
```

Comment:
```suggestion
                                    <li>We don’t screw you on contract terms, like auto-renewal.</li>
```

Is there something we can link to that explains this? Would "how we do sales?" cover this?